languages/fr/assess-api.json

```diff
@@ -86,16 +86,16 @@
     "plus": "+",
     "minus": "-",
     "zero": "0",
-    "zoomInResult": "Le navigateur effectuera un zoom avant lorsque vous appuyez sur la touche plus (+).",
-    "zoomOutResult": "Le navigateur diminue le zoom lorsque vous appuyez sur la touche Soustraction (-).",
+    "zoomInResult": "Le navigateur effectuera un zoom avant lorsque vous appuyez sur la touche Plus (+).",
```

Comment:
How about zoomer/ dé-zoomer?

Reply:
Don't know about this one. I find “dé-zoomer” a bit ugly, and “zoom avant/arrière” feels familiar.

Reply:
I used linguee.com a lot to try and find the most popular translations of some words, and zoom avant/arrière seemed to appear more often. Zoom doesn't even appear in the Académie Française's dictionary, so I assume zoom avant/arrière might be preferred in a school context.